src/Kinepolis/ParsedMovie.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Kinepolis;
+
+use CultuurNet\UDB3\Description;
```

Comment:
I would try to avoid using the legacy classes and only do the conversion from the new classes to the old classes where it is really needed. So in this case I would use the new description here and only do the conversion in the service. If we do the refactor of the `UpdateDescription` only one classes needs to be changed.